packages/blade/src/components/Input/PhoneNumberInput/_decisions/decisions.md

```diff
@@ -136,4 +136,24 @@ type PhoneNumberInputProps = CommonProps & {
 
 ## Open questions
 
-- How can we get the flag icons of all the countries?
+### MoM with i18n team
+
+**Participants:** RK, Anurag, Tarun
+
+**Concerns raised:**
+
+1. How to get the flag icons of all the countries?
+2. Should we format the phone number while typing?
+
+**Discussion points:**
+
+1. i18n team has exposed a function called [`getFlagOfCountry`](https://github.com/razorpay/i18nify/tree/master/packages/i18nify-js#getflagofcountrycountrycode-) which will return the flag svg via the [flagpedia CDN](https://flagpedia.net/download/api), we will have to render it as an image.
+2. Discussed about the legitimacy and security of flagpedia CDN.
+3. Formatting the phone number while typing will be a bit complicated on dev side and could cause edge cases. Thus as phase 1, we won't be formatting, Instead we will show the format in placeholder.
+4. For validation of phone number we will do it on client side and add `validationState` prop based on the result.
```

Comment:
Will we do it while user is typing? or on blur? and can it be standardised? because in a form, my email getting validated on submit button click and phone number getting validated while typing is strange no? 

What if consumer wants additional validation? can they still do it?

Reply:
true, i think given the concerns consumers can also do the validation on their end we don't need to handle it. 

const isPhoneValid = isValidPhoneNumber(...)
<PhoneNumberInput validationState={isPhoneValid ? 'error' : 'none'} />

Reply:
makes sense. this will also allow them to use it with their existing validation libraries